scripts/docs-extract.ts

```diff
@@ -0,0 +1,30 @@
+import * as path from 'path'
```

Comment:
This script may not be necessary since these same actions are performed during the build by the `extract-api` and `generate-plugin-schema` scripts

Reply:
Script removed.